### ISSUE TITLE
Body of the email sent by islandora via "contact collection admin" form contains rat html and unreadable

### ISSUE THREAD

**Issue by htpvu** [jhu-idc/iDC-general#487]
We need this to be human readable text. 
Resulted from this UAT https://docs.google.com/document/d/1_7RiiYoH5ip9z8ezJ9L0wJAdUgzpsg3QmpI2Ca94mwk/edit - link to usecase could also be found in this document 
![Screen Shot 2022-03-01 at 10 56 44 AM](https://user-images.githubusercontent.com/11050142/156202857-6c4e9943-deae-4b88-8e4c-f30b9374eac4.png)

**Comment by jabrah** [comment]
This sounds like it's a configuration issue with how the Drupal contact_emails module is being setup and used.

It seems like people have encountered this sort of issue before, see https://www.drupal.org/project/contact_emails/issues/3190740

I am not too sure how to fix this at the moment, but any fix we put into place would have to be applied to `Collection Contact` and `Repository item contact` contact forms in Drupal.

**Comment by DonRichards** [comment]
I Agree with John's comments, there's a setting in Drupal specifically for this. This isn't set up in the dev environment but probably should and disabled. I'm am not familiar enough with how dev & prod environments are provisioned to identify the exact course but this is doesn't prevent the site from functioning and stakeholders are experiencing limited or moderate issues with the site, so I'd rate this as a "Normal" priority. We should have a defined (if it doesn't already exist) priority matrix to help classify ticket priority. 

[Here](https://docs.google.com/document/d/1DzXD1nigBkhbnwSkAvaNuc0CWwNL0M9cidB2apB9sRk/edit?usp=sharing)'s a rough draft of a priority matrix I'm currently working off of.

**Comment by DonRichards** [comment]
I don't have the access to move this ticket to "In progress".
